### DISCUSSION TITLE
In-line compile errors do not show on correct line in the source member whenever program has copybooks

### DISCUSSION THREAD

**Discussion by cwbuck** [codefori/vscode-ibmi#1297] [General]
### Issue
At compile of an RPG program, if the program includes copy books, the compile listing interpreter has appeared to have issues for some time now trying to determine the actual line in which to show the in-line compile error for easy diagnosis. 
Even looking at the Problems, it does not seem reference the correct line in most cases. 

### Example
"Invalid token" caused by the double pipe used to concatenate here instead of the RPG "+" syntax. However, line issue is reported on 122 instead of here on 73 where the error actually occurs.
Command used to run in the action:
```
CRTSQLRPGI OBJ(&OPENLIB/&OPENMBR) 
  SRCFILE(&OPENLIB/&OPENSPF) 
  CLOSQLCSR(*ENDMOD) 
  OPTION(*EVENTF) 
  INCFILE(*LIBL/*SRCFILE) 
  RPGPPOPT(*LVL2) 
  TGTRLS(V7R4M0) 
  REPLACE(*YES) 
  SQLCURRULE(*DB2) 
  DBGVIEW(*SOURCE)
``` 
![image](https://github.com/halcyon-tech/vscode-ibmi/assets/68613130/906e181c-368a-491a-bdb8-e4c988a47512)

I remember reporting this issue a year or so ago and I think it got closed for a fix, but I'm not sure it was ever fully resolved. So before I report an issue, I'm just making sure there's not a config setting or compile parm or something I need to adjust. 
I've been trying to go all-in on this extension for a while, but I find myself having to revert back to RDi to reliably give me indication on things like these in-line errors.

**Comment by worksofliam** [comment]
@cwbuck Hey! Yes, there is an existing issue open for this: https://github.com/halcyon-tech/vscode-ibmi/issues/405

`*LVL2` adds some serious complexity to how source is mapped and the issue has never been fully solved. Sorry for the trouble - the quick work around is `*LVL1` I believe, though that's not an option for some.

**Reply by cwbuck** [reply]
Just found the old issue was still open a few minutes ago right before you sent. Thanks! I did just change back to *LVL1 and it worked. I just can't remember what the difference was and if it is acceptable for our situation or not.
